### PR TITLE
Add build pipeline for bootstrapper

### PR DESCRIPTION
**Pull request by Thahara**
## Purpose

Add a standalone build for the bootstrapper.

## Changes

Added Jenkinsfile, and related scripts.

## Checklist

- [ ] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: scripts/bootstrapper/build.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+CONCORDIUM_NODE_BOOTSTRAPPER_REGENESIS_BLOCK_HASHES_FILE=$p2p_bootstrapper_regenesis_block_hashes_file
+EOF
+
+cat <<EOF > DEBIAN/control
+Package: p2p-bootstrapper
+Version: $version
+Section: extra
+Priority: optional
+Architecture: amd64
+Depends: debhelper ( >= 10 ), libunbound2 ( >= 1.6.7 ) | libunbound8 ( >= 1.9.4 ), openssl ( >= 1.1.1 ), libgmp10 ( >= 6.1.2 ), libpq-dev ( >= 9.5.19)
```
Comment (abizjak) [inline scripts/bootstrapper/build.sh:52]: We don't need libunbound and openssl and libpq I think.

**Review by abizjak — APPROVED** [review]
I have not test run it, but it looks good overall.

If you can please check which dependencies are actually needed since the current list is quite old and obsolete.

File: scripts/bootstrapper/build.sh (hunk 1 above)
Comment (Thahara) [reply scripts/bootstrapper/build.sh]: Cleaned up the dependencies in accordance with `ldd` output.

File: scripts/bootstrapper/copy.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+rm -rf /out/*
+cp /build/*.deb /out
+
+#TODO: Fix permissions on output
+#chown -R $EXTERNAL_UID:$EXTERNAL_GID /out
```
[the remark this answers is not in the document]
Comment (Thahara) [reply scripts/bootstrapper/copy.sh]: Implemented the ownership change, so any build artifacts moved out of the docker container is owned by the Jenkins user.

**Pull request merged**